<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_task_widget</class>
 <widget class="QWidget" name="add_task_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="project_list"/>
   </item>
   <item>
    <widget class="QTextEdit" name="task_text_edit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_h_layout">
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
